models/ChatbotMessage.ts

```diff
@@ -0,0 +1,10 @@
+const ChatbotMessageSchema = new Schema({
+  body: { type: String, required: true },
+  images: { type: Array, required: false },
+  nextMessages: { type: Array, required: true },
+  previousMessage: { type: String, required: true },
```

Comment:
Type this as a reference to the key of another ChatbotMessage, same as explained above